libs/ml/src/core/graph.cpp

```diff
@@ -1111,6 +1111,21 @@ std::vector<std::pair<std::string, std::vector<std::string>>> Graph<TensorType>:
   return connections_;
 }
 
+template <typename TensorType>
+fetch::ml::MLChargeAmount Graph<TensorType>::ChargeForward(const std::string &node_name)
+{
+  if (nodes_.find(node_name) == nodes_.end())
+  {
+    FETCH_LOG_ERROR(DESCRIPTOR, "The graph does not contain a node with name " + node_name);
+    return 0;
+  }
```

Comment:
this won't catch cases where I want to call forward from the node in a subgraph. the node name wont be present in graph.nodes_; I think you want to use @MissingNO57's new functionality for finding a node in a graph - this makes a recursive call checking all subgraphs.

Reply:
For now it is intended to work with most parent Graph only; assuming we have FC and Conv layers as subgraphs, their froward pass charge estimation is calculated separately inside corresponding Layer code; and while in parent (model) graph, any Node is accessible.